src/pages/find-offers/FindOffers.jsx

```diff
@@ -50,10 +56,14 @@ const FindOffers = () => {
           open={ openFilters }
           showingTutorOffers={ showingTutorOffers }
         /> 
-        <OfferCard
-          offer={ mockOffer }
-          onBookmarkClick={ () => onBookmarkClick(mockOffer.id) }
-        />
+        <Box>
+          { currentOffersOnPage.map(el => (<OfferCard
+            key={ el.id }
+            offer={ el }
+            onBookmarkClick={ () => onBookmarkClick(el.id) }
+            // eslint-disable-next-line react/jsx-closing-bracket-location
+          />)) }
```

Comment:
Move out it from jsx and maybe doesn't need to leave comment for eslint. I think it pushes without any problems.

Reply:
fixed